src/types/rpc/router-rpc.ts

```diff
@@ -34,28 +36,35 @@ export enum FailureCode {
   PERMANENT_NODE_FAILURE = 20,
   PERMANENT_CHANNEL_FAILURE = 21,
   EXPIRY_TOO_FAR = 22,
+  MPP_TIMEOUT = 23,
   UNKNOWN_FAILURE = 998,
   UNREADABLE_FAILURE = 999,
 }
 
 export interface SendPaymentRequest {
   dest?: Buffer | string;
   amt?: number;
+  amtMsat?: number;
   paymentHash?: Buffer | string;
   finalCltvDelta?: number;
   paymentRequest?: string;
   timeoutSeconds?: number;
   feeLimitSat?: number;
-  outgoingChanId?: number;
+  feeLimitMsat?: number;
+  outgoingChanId?: string;
+  lastHopPubkey?: Buffer | string;
   cltvLimit?: number;
   routeHints?: RouteHint[];
-  destTlvMap?: Array<[number, Buffer | string]>;
+  destCustomRecordsMap?: Array<[number, Uint8Array]> | string[];
+  allowSelfPayment?: boolean;
+  destFeatures?: FeatureBit[];
 }
 
-export interface PaymentStatusUpdate {
+export interface PaymentStatus {
```

Comment:
This was called `PaymentStatusUpdate` to avoid changing the `PaymentStatus` enum name